spec/openapi.yaml

```diff
@@ -522,20 +522,30 @@ paths:
             application/json:
               schema:
                 oneOf:
-                  - $ref: "#/components/schemas/PythonPackageUpdateInfo"
+                  - $ref: "#/components/schemas/PrusaLinkPackage"
         204:
           description: No Content
+          headers:
+            Update-Available:
```

Comment:
204 mean, no update available right?!

Reply:
Yes, if there's no update available, this header has value "false".
I suppose you mean this header should not be here at all? I can change it.